devops/clean

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Cleans all build artifacts and custom config from the repo.
+# Intended for use on developer workstations. Should NOT be run
+# on production Admin Workstations.
+set -e
```

Comment:
This `set -e` doesn't propagate to the `remove_unwanted_files` function which is probably where we want it?

Reply:
Oh, but it does!

```
$ git diff
diff --git a/devops/clean b/devops/clean
index 6ccdbb8ff..99243c3f3 100755
--- a/devops/clean
+++ b/devops/clean
@@ -8,6 +8,11 @@ set -u
 
 function remove_unwanted_files() {
 
+    echo "BEFORE FALSE"
+    false
+    echo "AFTER FALSE"
+    exit 5
+
     # Remove any Onion URL info from previous instances.
     rm -vf install_files/ansible-base/app-source-ths \
         install_files/ansible-base/app-ssh-aths \
 $ make clean
WARNING: Proceeding will destroy all local customizations, including
application configuration, user credentials, Onion URLs, static assets,
VMs, and all git-ignored files. It is NOT possible to undo these changes!
To proceed, type 'yes': yes
BEFORE FALSE
Makefile:152: recipe for target 'clean' failed
make: *** [clean] Error 1
```

The "AFTER FALSE" message never appears, due to `set -e` halting execution of the script.

Reply:
Wow, I learned that wrong ages ago and just never relied on it in functions. Well never mind then :)